app/src/NativeBase/Components/Forms/TextInputControl.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file Form input.
```

Comment:
Maybe say it's a Text input with error message etc?  Just to make it clearer here